_posts/2018/2018-02-05-enumerable-avoids-using-temporary-variables.md

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Enumerable avoids using temporary variables when looping'
+description: 'Idiomatic Ruby is in the looping'
+layout: article
+category: ruby
+image:
+  base: '2018/enumerable-avoids-using-temporary-variables'
+  alt: 'Fruit loops'
+  source: 'https://unsplash.com/photos/BumOnw4oEZo'
+  credit: 'David Streit'
+---
+
+Some of my favourite Ruby features are to be found in the Enumerable module. You can read more in the [Ruby documentation](http://ruby-doc.org/core-2.5.0/Enumerable.html).
+
+A class representing a collection, such as Array, Set or Hash, has the methods and features of Enumerable included. `Enumerable` contains methods to allow you to loop through the members of that group and 'do stuff' based on each member of the collection.
```

Comment:
*'do stuff' to each member of the collection

Reply:
Not necessarily ‘to’ actually.

Reply:
“You take an action on every element of the collection” ?

Reply:
Yes, was thinking this..about the 'to' thing...

Perhaps: *take an action with respect to each element.. or 'relative to'...

"on" still has the same connotations as "to", no?